lib/bottom_navy_bar.dart

```diff
@@ -121,7 +158,7 @@ class _ItemWidget extends StatelessWidget {
         curve: curve,
         decoration: BoxDecoration(
           color:
-              isSelected ? item.activeColor.withOpacity(0.2) : backgroundColor,
+              isSelected ? item.activeColor.withOpacity(0.1) : backgroundColor,
```

Comment:
It's better to have `activeOpacity` as constructor option.

Reply:
Hi @pawelpetruch
I agree with @omidraha it will be better to have this as constructor argument

Reply:
Sure, I'm moving it to the constructor.